codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -3,27 +3,34 @@
 // found in the LICENSE file.
 
 import 'dart:async';
-import 'package:codesign/codesign.dart';
 import 'package:file/file.dart';
 import 'package:process/process.dart';
-import 'package:file/local.dart';
+import 'package:logging/logging.dart';
 import 'dart:io' as io;
 
-/// Interface for classes that interact with files nested inside of [RemoteZip]s.
-abstract class FileVisitor {
-  const FileVisitor();
-
-  // TODO(xilaizhang): add back the visit interfaces
-}
-
+/// The notarization status reported by notary server, after user uploaded
```

Comment:
```dart
/// Statuses reported by Apple's Notary Server.
///
/// See more:
///   * https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution/customizing_the_notarization_workflow
```

I'm not sure this is the right URL, but linking to the original documentation is a helpful breadcrumb

Reply:
Good advice! Changed.